modules/virt-hot-plugging-a-virtual-disk-using-the-web-console.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc
+
+[id="virt-hot-plugging-a-virtual-disk-using-the-web-console{context}"]
+= Hot-plugging a virtual disk using the web console
+
+Hot-plug virtual disks that you want to attach to a virtual machine instance (VMI) while a virtual machine is running.
+
+.Prerequisites
+* You must have a running virtual machine to hot-plug a virtual disk.
+* You must have at least one data volume or persistent volume claim (PVC) available for hot-plugging.
```

Comment:
UI creates the data volume or PVC by adding the disk, so this is not a prerequisites for UI.
Plus, don't see data volume or PVC mentioned in the procedure, so this is not needed.